app/views/template_sync_settings/show.json.rabl

```diff
@@ -1,11 +1,11 @@
 object @setting
 
 node do |setting|
-  { :name => setting.short_name }
+  { :name => setting.name.delete_prefix('template_sync_') }
 end
 
 attributes :id, :value, :description, :settings_type, :default, :full_name
 
 node do |setting|
-  { :selection => setting.selection }
+  { :selection => (setting.select_values&.map { |key, label| { value: key, label: label } } || []) }
```

Comment:
In general I prefer providing default values as soon as possible, to me it is easier to read.
```
(setting.select_values || []).map { |k, v| ... }
```

As this is a style-only change, feel free to ignore this comment